s3_scrubber/src/lib.rs

```diff
@@ -140,7 +140,6 @@ pub fn init_logging(binary_name: &str, dry_run: bool, node_kind: &str) -> Worker
         .with_writer(file_writer);
     let stdout_logs = fmt::Layer::new()
         .with_target(false)
-        .with_ansi(atty::is(atty::Stream::Stdout))
```

Comment:
Could we maybe still do terminal detection e.g. by using the [`IsTerminal`](https://doc.rust-lang.org/stable/std/io/trait.IsTerminal.html) trait?

Reply:
perhaps, but it's easy to add that back in if we decide that we care about having colored output from the scrubber (personally I'm always piping output to a file)

Reply:
This was a one liner so I just added IsTerminal.